dock/plugins/pre_inject_yum_repo.py

```diff
@@ -34,12 +34,12 @@ def add_yum_repos_to_dockerfile(yumrepos, df):
     preinsert = None
     structure = df.structure
     for insndesc in structure:
-        insn = insndesc['instruction'].lower()
-        if insn == 'maintainer':
+        insn = insndesc['instruction']
+        if insn == 'MAINTAINER':
```

Comment:
Yes, `insndesc['instruction']` is changed to upper-case in `Dockerfile.structure()` now.